dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/exceptions/ServiceExceptionV2.java

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Maybe this could be put into the package `org.apache.dolphinscheduler.api.v2.exceptions`